### DISCUSSION TITLE
Save data for cluster faces

### DISCUSSION THREAD

**Discussion by JeanCGR** [nextcloud/recognize#811] [Q&A]
Hi guys,

I have an nextcloud version 26.0.0 with recognize application.
The people present on the photos have been detected and assembled in the "person" tab but I would like to save the data on the cluster faces.

Do you know where this data is stored? Is it in a file?

Thank you guys

**Comment by marcelklehr** [comment]
> I would like to save the data on the cluster faces.
> Do you know where this data is stored? Is it in a file?

The information which face is assigned to which cluster is stored in the database.

**Reply by marcelklehr** [reply]
You can also retrieve it using webdav at /remote.php/dav/recognize
